core/sdk-core/src/main/java/software/amazon/awssdk/core/SplitTransformerConfiguration.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core;
+
+import java.util.Objects;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.core.async.AsyncResponseTransformer;
+import software.amazon.awssdk.core.internal.async.SplittingTransformer;
+import software.amazon.awssdk.utils.Validate;
+import software.amazon.awssdk.utils.builder.CopyableBuilder;
+import software.amazon.awssdk.utils.builder.ToCopyableBuilder;
+
+/**
+ * Configuration options for {@link AsyncResponseTransformer#split(SplitTransformerConfiguration)} to configure how the SDK
+ * should split the {@link AsyncResponseTransformer}.
+ *
+ * @see #builder()
+ */
+@SdkPublicApi
+public final class SplitTransformerConfiguration implements ToCopyableBuilder<SplitTransformerConfiguration.Builder,
+    SplitTransformerConfiguration> {
+
+    private final Long bufferSize;
+
+    private SplitTransformerConfiguration(DefaultBuilder builder) {
+        this.bufferSize = Validate.paramNotNull(builder.bufferSize, "bufferSize");
+    }
+
+    /**
+     * Create a {@link FileRequestBodyConfiguration.Builder}, used to create a {@link SplitTransformerConfiguration}.
```

Comment:
Why does this mention `FileRequestBodyConfiguration`?

Reply:
oops missed deleting that